observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -12,6 +12,73 @@ The Network Observability Operator enables administrators to observe and analyze
 These release notes track the development of the Network Observability Operator in the {product-title}.
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
+[id="network-observability-operator-release-notes-1-7_{context}"]
+== Network Observability Operator 1.7.0
+The following advisory is available for the Network Observability Operator 1.7.0:
+
+* link:https://access.redhat.com/errata/RHSA-2024:8014[Network Observability Operator 1.7.0]
+
+[id="network-observability-operator-1.7.0-features-enhancements_{context}"]
+=== New features and enhancements
+
+[id="network-observability-operator-otel-1-7_{context}"]
+==== OpenTelemetry support
+You can now export enriched network flows to a compatible OpenTelemetry endpoint, such as the Red Hat build of OpenTelemetry. For more information see xref:../../observability/network_observability/configuring-operator.adoc#network-observability-enriched-flows_network_observability[Export enriched network flow data].
+
+[id="network-observability-operator-developer-perspective-1-7_{context}"]
+==== Network Observability Developer perspective
+You can now use Network Observability in the *Developer* perspective. For more information, see xref:../../observability/network_observability/network-observability-overview.adoc#no-console-integration[{product-title} console integration]
+
+[id="network-observability-operator-tcp-flags-filtering-1-7_{context}"]
+==== TCP flags filtering
+You can now use the `tcpFlags` filter to control the volume of packets cached in the eBPF flow table. For more information, see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-flowcollector-flowfilter-parameters_nw-observe-network-traffic[Flow filter configuration parameters] and xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-ebpf-flow-rule-filter_nw-observe-network-traffic[eBPF flow rule filter].
```

Comment:
I would rather say "to limit the volume of packets processed by the eBPF program", because there's more than caching, and I guess readers don't have to know about the internal details

```suggestion
You can now use the `tcpFlags` filter to control the volume of packets processed by the eBPF program. For more information, see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-flowcollector-flowfilter-parameters_nw-observe-network-traffic[Flow filter configuration parameters] and xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-ebpf-flow-rule-filter_nw-observe-network-traffic[eBPF flow rule filter].
```